frb_codegen/src/generator/rust/ty.rs

```diff
@@ -49,6 +49,22 @@ pub trait TypeRustGeneratorTrait {
         Acc::default()
     }
 
+    fn deallocate_funcs(
+        &self,
+        _collector: &mut ExternFuncCollector,
+        _block_index: BlockIndex,
+    ) -> Acc<Option<String>> {
+        Acc::default()
+    }
+
+    fn opaque_related_funcs(
```

Comment:
Hmm this means all types will have to be aware of opaque. Is it possible to avoid this?